ddp-workspace/projects/ddp-testboston/src/style/right-to-left.scss

```diff
@@ -0,0 +1,280 @@
+html[dir='rtl'] {
+
```

Comment:
Do we need to set this attribute in the DOM ourselves? (I am not sure. This is more of question).
I tried setting it in Chrome after switching to Arabic using the language selector.
It does set the text alignment to the right and the cursor for input moves right to left as you enter text.
If we are not setting it, when does this style get applied?

Reply:
Also home page looks better when setting this attribute. Obviously you counting on this as CSS selector. How do we set it?

Reply:
> Do we need to set this attribute in the DOM ourselves? (I am not sure. This is more of question).
I tried setting it in Chrome after switching to Arabic using the language selector.
It does set the text alignment to the right and the cursor for input moves right to left as you enter text.
If we are not setting it, when does this style get applied?

We absolutely need to set up this attribute, because ltr languages should be mirrored on the screen. 

>Also home page looks better when setting this attribute. Obviously you counting on this as CSS selector. How do we set it?

We set i18n attributes here: https://github.com/broadinstitute/ddp-angular/blob/DDP-5179_rtl/ddp-workspace/projects/ddp-sdk/src/lib/components/internationalization/languageSelector.component.ts#L252